freqtrade/tests/test_freqtradebot.py

```diff
@@ -785,14 +784,15 @@ def test_check_handle_timedout_buy(default_conf, ticker, limit_buy_order_old, mo
         validate_pairs=MagicMock(),
         get_ticker=ticker,
         get_order=MagicMock(return_value=limit_buy_order_old),
-        cancel_order=cancel_order_mock
+        cancel_order=cancel_order_mock,
+        get_fee=MagicMock(return_value=0.0025)
     )
     freqtrade = FreqtradeBot(default_conf, create_engine('sqlite://'))
 
     trade_buy = Trade(
         pair='ETH/BTC',
         open_rate=0.00001099,
-        exchange='BITTREX',
+        exchange='bittrex',
```

Comment:
I think we should standardize on one way to name exchanges (lower / upper / initcap) ... `Binance` was used in another test, now it's `bittrex` - shouldn't the format be consistent?

Reply:
Lowercase (ID, from get_id() ) should be used for referencing in the code and config. Mixed case should be used for printing (name, get_name() )

Reply:
ah ok, wasn't aware of that (we should document that somewhere i guess), makes sense to me